mlflow/sagemaker/__init__.py

```diff
@@ -126,16 +126,15 @@ def push_image_to_ecr(image=DEFAULT_IMAGE_NAME):
     # x = ecr_client.get_authorization_token()['authorizationData'][0]
     # docker_login_cmd = "docker login -u AWS -p {token} {url}".format(token=x['authorizationToken']
     #                                                                ,url=x['proxyEndpoint'])
+
+    docker_login_cmd = "aws ecr get-login-password" \
+                       " | docker login  --username AWS " \
+                       " --password-stdin " \
+                       "{account}.dkr.ecr.{region}.amazonaws.com".format(
+                            account=account,
+                            region=region)
+
     os_command_separator = ";\n"
-    if platform.system() == "Windows":
-        os_command_separator = " && "
```

Comment:
I think we still need the ` && ` OS command separator for windows, right?

Reply:
Yes, you're right, I'll add it back.